### PR TITLE
Fix/proofread

### PR DESCRIPTION
**Pull request by patrickvacek**
I have not actually proofread all the deployment docs yet. This is just what I found with a a quick scan and lots of grepping.

### CODE REVIEW

**Review by jhdalek55 — APPROVED** [review]

**Review by jhdalek55 — APPROVED** [review]

File: README.md
Comment (jhdalek55) [inline README.md:8]: Since this commit is devoted to capitalization, please also capitalize  both Deployment Best Practices and Github.

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
Approved except for one missed capitalization

**Review by jhdalek55 — APPROVED** [review]

**Review by jhdalek55 — APPROVED** [review]

**Review by jhdalek55 — APPROVED** [review]

**Comment by jhdalek55** [comment]
I've decided I should wait till we get your PRs merged before I do a pass through. I may read it offline and mark it up that way, but since we may address some of the same issues, I think its better not to have overlapping PRs.

File: README.md
Comment (patrickvacek) [reply README.md]: Good call, fixed.

**Comment by patrickvacek** [comment]
I'll stop adding commits to this for now so we can merge it as is if someone with merge rights wants to press the buttons.

**Review by trishankatdatadog — APPROVED** [review]

**Pull request merged**
